File: PDOO/NapakalakiRuby.ruby/lib/prize.rb
#encoding: utf-8

module NapakalakiGame

  class Prize

    attr_reader :treasures, :level

    public  
    
    def initialize(treasures, levels)
      @treasures = treasures
      @level = levels
    end
    
    def getTreasures
      return @treasures
    end
    
    def getLevels
      return @level
    end

    def to_s
      "\n\tTesoros ganados: #{@treasures}\n\tNiveles ganados: #{@level}"
    end
  end
end

File: PDOO/Napakalaki.java/src/NapakalakiGame/Prize.java
package NapakalakiGame;

/**
 * @author Andres Arco Lopez
 */
public class Prize {
    private int treasures;
   
    private int level;
    
    public Prize(int tr, int lv){
        treasures=tr;
        level=lv;
    }
    
    public int getTreasures(){
        return treasures; 
    }
    
    public int getLevel(){
        return level;
    }
    
    public String toString(){ 
        return "\n\tTesoros= " +Integer.toString(treasures) + "\n\tNiveles= "+ Integer.toString(level);
    }
}
